syft/execution/action.py

```diff
@@ -10,4 +20,258 @@ class Action(ABC):
     tensors and add them together is an Action. Sending an object from one worker to another is
     also an Action."""
 
-    pass
+    def __init__(
+        self, name: str, target, args_: tuple, kwargs_: dict, return_ids: tuple, return_value=False
+    ):
+        """Initialize an action
+
+        Args:
+            name (String): The name of the method to be invoked (e.g. "__add__")
+            target (Tensor): The object to invoke the method on
```

Comment:
Is it necessarily a tensor here or can it be a Syft object as well?

Reply:
I'm not sure. In concept, it certainly could be (and should be) a Syft object, but it may only work for tensors right now.

Reply:
Ok, we keep it as is for now then 👍